helpdesk_stock/views/stock_request_order.xml

```diff
@@ -26,17 +26,4 @@
         </field>
     </record>
 
-    <record id="stock_request_order_helpdesk_ticket_search" model="ir.ui.view">
-        <field name="model">stock.request.order</field>
-        <field name="inherit_id" ref="stock_request_picking_type.view_stock_request_order_filter"/>
-        <field name="arch" type="xml">
-            <field name="name" position="after">
-                <field name="helpdesk_ticket_id"/>
-            </field>
-            <filter name="state" position="before">
-                <filter string="Helpdesk Ticket" name="helpdesk_ticket_id" domain="[]" context="{'group_by': 'helpdesk_ticket_id'}"/>
-            </filter>
-        </field>
-    </record>
-
```

Comment:
@mtelahun Thanks for the contribution. Can we keep this view on the stock request order?

Reply:
There is no stock request order in Inventory view. Only stock request views are created by stock_request_picking_type.